free-programming-books.md

```diff
@@ -2439,6 +2440,7 @@ Kerridge (PDF) (email address *requested*, not required)
 
 ### Raspberry Pi
 
+* [Raspberry PI Books](https://magpi.raspberrypi.org/books) - The official Raspberry Pi magazine (PDF) (meta-list)
```

Comment:
we don't include lists of books in our lists of books.

Reply:
So should we not include the Raspberry Pi books at all mentioned on the issue, or do I need to add them on one by one basis?

Reply:
@eshellman I removed the list, let me know if I should add the books one by one.